lib/meta_inspector/parsers/base.rb

```diff
@@ -25,6 +25,18 @@ def initialize(main_parser)
       def cleanup(results)
         results.map { |r| r.value.strip }.reject(&:empty?).uniq
       end
+
+      def sanitize(raw_html)
+        MetaInspector.sanitizer.sanitize(raw_html)
+      end
+
+      def self.sanitized_attributes(*att_names)
+        att_names.each do | attribute |
```

Comment:
`attribute_names` instead of `att_names` might be slightly more readable and only a bit longer.